### PR TITLE
EZP-31027: Disabled language is available in COTF

### PR DESCRIPTION
**Pull request by konradoboza**
JIRA ticket: https://jira.ez.no/browse/EZP-31027

Targets to 2.5 only, to 3.0 there will be another PR.

### CODE REVIEW

File: src/modules/universal-discovery/components/create/choose.language.component.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default class ChooseLanguageComponent extends Component {
             attrs.selected = true;
         }
 
-        return <option {...attrs}>{language.name}</option>;
+        return language.enabled ? <option {...attrs}>{language.name}</option> : null;
```
Comment (dew326) [inline src/modules/universal-discovery/components/create/choose.language.component.js]: Please make an early return, it's more readable.

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/modules/universal-discovery/components/create/choose.language.component.js (hunk 1 above)
Comment (konradoboza) [reply src/modules/universal-discovery/components/create/choose.language.component.js]: Changed in https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/252/commits/65e5e93880c0377e2d4ffef45d5abb77d8d1b0e7.

**Review by GrabowskiM — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by OstafinL — APPROVED** [review]

**Review by micszo — APPROVED** [review]
QA Approved on eZ Platform EE 2.5 with diff.

**Comment by konradoboza** [comment]
ping @lserwatka

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by konradoboza** [comment]
@lserwatka done.

**Pull request merged**